browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4064,6 +4064,39 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       Up next
     </message>
     <!-- End of Playlist strings -->
+    <message name="IDS_LEO_UNLEASH_POWER" desc="Title for Brave Leo subscription purchase">
+      Unleash Leo's full powers with Premium:
+    </message>
+    <message name="IDS_LEO_TIP_EXPLORE_TITLE" desc="Title for Brave Leo subscription Explore tip">
+      Explore different AI models
+    </message>
+    <message name="IDS_LEO_TIP_EXPLORE_DESCRIPTION" desc="Description for Brave Leo subscription Explore tip">
+      Priority access to powerful models with different skills.
+    </message>
+    <message name="IDS_LEO_TIP_UNLOCK_TITLE" desc="Title for Brave Leo subscription Unlock tip">
+      Unlock your creativity
+    </message>
+    <message name="IDS_LEO_TIP_UNLOCK_DESCRIPTION" desc="Description for Brave Leo subscription Unlock tip">
+      Access models better suited for creative tasks and content generation.
+    </message>
+    <message name="IDS_LEO_TIP_STAY_TITLE" desc="Title for Brave Leo subscription Stay on topic tip">
+      Stay on topic
+    </message>
+    <message name="IDS_LEO_TIP_STAY_DESCRIPTION" desc="Description for Brave Leo subscription Stay on topic tip">
+      Get more accurate answers for more nuanced conversations.
+    </message>
+    <message name="IDS_LEO_TIP_CHAT_TITLE" desc="Title for Brave Leo subscription Chat for longer tip">
+      Chat for longer
+    </message>
+    <message name="IDS_LEO_TIP_CHAT_DESCRIPTION" desc="Description for Brave Leo subscription Chat for longer tip">
+      Get higher rate limits for longer conversations.
+    </message>
+    <message name="IDS_LEO_SUBSCRIPTION_DISCLAIMER" desc="Disclaimer for Brave Leo subscription">
+      All subscriptions are auto-renewed but can be\ncancelled at any time before renewal.
```

Comment:
is it ok to have `\n` here? I don't remember we used this before.

Reply:
yeah we should be okay. i have tried it in the past with rewards texts.